modules/analyzing-all-violations.adoc

```diff
@@ -11,6 +11,10 @@ By viewing the *Violations* page, you can analyze all violations and take correc
 .Procedure
 
 . In the {product-title-short} portal, click *Violations*.
+. To navigate through the different types of violations, do any of the following tasks:
```

Comment:
```suggestion
. To navigate through the different types of violations, perform any of the following tasks:
```

Reply:
Thanks for the suggestion, however, I would prefer to keep it unchanged to maintain consistency across ACS docs. I don't think there is any style guide restriction on the usage of "do".